ic-os/guestos/context/Dockerfile

```diff
@@ -111,7 +111,8 @@ RUN systemctl disable \
     motd-news.service \
     motd-news.timer \
     fstrim.service \
-    fstrim.timer
+    fstrim.timer \
+    filebeat.service
```

Comment:
The following can also be done in a follow up,
but if you're disabling filebeat we should also disable configuring it in the system-test-driver. Basically you want to remove [all these occurrences](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/dfinity/ic%24+file:rs/tests/driver+elasticsearch+&patternType=keyword&sm=0) (except for the matches in `rs/tests/driver/src/driver/vector_vm.rs` of course).

Reply:
I will revert that change, it was just for testing. This should be done in a follow up